packages/Textarea/src/Textarea.js

```diff
@@ -37,56 +37,64 @@ const defaultProps = {
 };
 
 function Textarea(props) {
-  let textarea = null;
+  const textarea = React.useRef(null);
```

Comment:
I feel like I've seen people suffixing ref names with `Ref` elsewhere (so `textareaRef`) although could be wrong

Reply:
you could be right! :) changed